src/commands/power.rb

```diff
@@ -20,12 +22,35 @@
 # https://github.com/alces-software/metalware
 #==============================================================================
 
-#NB Settings in this file may be overridden by the shared or local bmc configs
-###########################
-#USERNAME=admin
-#PASSWORD=password
-###########################
+require 'commands/ipmi'
+
+module Metalware
+  module Commands
+    class Power < Ipmi
+      private
 
-#IPMI_HEAD=
-IPMI_TAIL=.bmc
-STEP_INTERVAL=2
+      def render_command
+        case args[1].to_s
+        when 'on'
+          'chassis power on'
+        when 'off'
+          'chassis power off'
+        when 'locate'
+          'chassis identify force'
+        when 'locateoff'
+          'chassis identify 0'
+        when 'status'
+          'chassis power status'
+        when 'cycle'
+          'chassis power cycle'
+        when 'reset'
+          'chassis power reset'
+        when 'sensor'
+          'sensor'
+        else
+          raise MetalwareError "Invalid power command: #{cmd}"
```

Comment:
Does this work? I don't think it will:

```ruby
[1] pry(main)> raise StandardError "foo"
NoMethodError: undefined method `StandardError' for main:Object
from (pry):1:in `__pry__'
```

I think you're missing a comma after `MetalwareError`.

Reply:
You are correct, fixed - thanks! :)